src/+otp/Problem.m

```diff
@@ -268,5 +281,13 @@ function validateNewState(obj, newTimeSpan, newY0, newParameters)
                 error('Expected solution to have %d variables but has %d', obj.NumVars, n);
             end
         end
+        
+        function t = parseTime(obj, t)
+            if ~isvector(t) || isempty(t) || ~otp.utils.validation.isNumerical(t)
+                error('The times must be a nonempty vector of numbers');
```

Comment:
I don't like the naming of t, maybe 'The input argument must be...'

Reply:
Not sure I understand. This function also gets called when solution struct (and not explicit t argument) is passed to `problem.solve`.

Reply:
Hmmm, so it is multi-purpose. OK that makes it harder to change. It is fine for now.